insights/client/phase/v1.py

```diff
@@ -127,13 +127,66 @@ def post_update(client, config):
     # create a machine id first thing. we'll need it for all uploads
     logger.debug('Machine ID: %s', client.get_machine_id())
     logger.debug("CONFIG: %s", config)
+
+    # -------delete everything below this line-------
```

Comment:
Is everything below this line being deleted later?

Reply:
yep. once the legacy path is phased out.